package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This is a good start, there are a couple changes you can make to improve the file.  Additionally, this file should be located within your lab assignment folder.